tests/dcerpc/dcerpc-dce-stub-data/test.yaml

```diff
@@ -0,0 +1,32 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 2
+    match:
+      event_type: dcerpc
```

Comment:
We'd need a min-version check for 6 here since dcerpc logging wasn't supported before that.